Bottom: 1346be7e8d8b9bed9b65f4740d3dd00f00540b81
Top:    fce7f5b1ad9fc45b141a4b75dd97232f90c5a2a8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-27 04:54:20 +0100

Refresh of shiny-3d-eye-candy-0

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 1c7d2a6..46b368c 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -105,8 +105,8 @@ compute_depth (int group)
   int middle_copper_group;
   int depth;
 
-  bottom_group = GetLayerGroupNumberByNumber (BOTTOM_SIDE);
-  top_group = GetLayerGroupNumberByNumber (TOP_SIDE);
+  bottom_group = GetLayerGroupNumberBySide (BOTTOM_SIDE);
+  top_group = GetLayerGroupNumberBySide (TOP_SIDE);
 
   min_copper_group = MIN (bottom_group, top_group);
   max_copper_group = MAX (bottom_group, top_group);
